armsrc/iso14443a.c

```diff
@@ -2091,7 +2123,11 @@ void ReaderIso14443a(UsbCommand *c)
 				ReaderTransmit(cmd,len, NULL);											// 8 bits, odd parity
 			}
 		}
+		//start_ts = GetCountSspClk(); //started just after we send all our bytes to the PICC
 		arg0 = ReaderReceive(buf, par);
+		//end_ts = GetCountSspClk(); //ended just after we have received all the response bytes from the PICC.
+		//uint32_t cycle_count = end_ts - start_ts;
+		//Dbprintf("Cycle count (all bytes) = %d", cycle_count);
```

Comment:
In general, none of these changes are necessary at this point in time. I propose to leave the file as it is.